src/include/btree.i

```diff
@@ -970,7 +970,7 @@ __wt_page_can_evict(WT_SESSION_IMPL *session, WT_PAGE *page, int check_splits)
 	 * a transaction value, once that's globally visible, we know we can
 	 * evict the created page.
 	 */
-	if (WT_PAGE_IS_INTERNAL(page) &&
+	if (check_splits && WT_PAGE_IS_INTERNAL(page) &&
```

Comment:
`mod_split_txn` is only set when we deepen the tree, and the child pages we create when we deepen the tree dirty the tree, so I don't think this is necessary.